tesseract/tesseract_kinematics/include/tesseract_kinematics/core/validate.h

```diff
@@ -58,40 +58,85 @@ inline bool checkKinematics(const tesseract_kinematics::ForwardKinematics::Const
 {
   // Check name
   if (fwd_kin->getName() != inv_kin->getName())
+  {
+    CONSOLE_BRIDGE_logError("checkKinematics: Manipulator names do not match!");
     return false;
+  }
 
   // Check if number of joints
   if (fwd_kin->numJoints() != inv_kin->numJoints())
+  {
+    CONSOLE_BRIDGE_logError("checkKinematics: Manipulator number of joints do not match!");
```

Comment:
Maybe list them for convenience?
`CONSOLE_BRIDGE_logError("checkKinematics: Manipulator number of joints do not match! Fwd: %d, Inv: %d", fwd_kin->numJoints(), inv_kin->numJoints() );`

Reply:
Updated